features/support/stub_calculator_requests.rb

```diff
@@ -0,0 +1,6 @@
+require 'webmock/cucumber'
+
+Before('@stub_calculator_request_and_fail') do
+  stub_request(:get, %r{\Ahttps://laa-fee-calculator(.*).k8s.integration.dsd.io/api/v1/.*\z}).
```

Comment:
**[not a blocker]** If possible, it would be better to use the host configuration (presumably configured in the ENV) rather than statically defining here, no?